packages/keystone/src/lib/server/createExpressServer.ts

```diff
@@ -35,7 +35,13 @@ const addApolloServer = ({
   server.use(graphqlUploadExpress({ maxFileSize }));
   // FIXME: Support custom API path via config.graphql.path.
   // Note: Core keystone uses '/admin/api' as the default.
-  apolloServer.applyMiddleware({ app: server, path: '/api/graphql', cors: false });
+  await apolloServer.start();
+  apolloServer.applyMiddleware({
+    app: server,
+    path: '/api/graphql',
+    // FIXME: Allow this to be set through config
+    cors: { origin: 'https://studio.apollographql.com', credentials: true },
```

Comment:
Could we not allow this in production?

Not really a thing to action but it's kinda annoying this is on a different origin to the API which means you can't sign in to the Admin UI and then use that logged in state while exploring the GraphQL API, you'd have to do the mutation to sign in 😞